### DISCUSSION TITLE
(build problems) I can't install cmake.

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#1650] [Q&A]
apparently I need CMAKE to build a tic-80 executable.

but believe it!!!

I searched the ENTIRE internet, and I couldn't find a solution.

**Comment by RobLoach** [comment]
https://github.com/nesbox/TIC-80#build-instructions

**Comment by bettywolfy** [comment]
i've been on the website (cmake.org) i just can't download and install.

**Reply by RobLoach** [reply]
I assume you're using Windows? Try the Visual Studio build instead? CMake is at https://cmake.org/download/ , correct.

**Reply by bettywolfy** [reply]
I don't have visual studio code it's too big, but I'll think about it.

**Reply by bettywolfy** [reply]
I got it!!!!!

Thanks dude!
